spec/controllers/users/email_confirmations_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe Users::EmailConfirmationsController do
+  describe '#create' do
+    describe 'Valid email confirmation tokens' do
+      it 'tracks a valid email confirmation token event' do
+        user = create(:user)
+        new_email = Faker::Internet.safe_email
+        add_email_form = AddUserEmailForm.new
+        add_email_form.submit(user, email: new_email)
+        email_record = add_email_form.email_address_record(new_email)
+
+        get :create, params: { confirmation_token: email_record.reload.confirmation_token }
+
+        expect(PushNotification::TestEventQueue.events).to eq [
```

Comment:
rspec does not seem to allow for multiple expect/receive if there are different arguments? not sure on the best way to handle this

```ruby
  1) Users::EmailConfirmationsController#create Valid email confirmation tokens tracks a valid email confirmation token event
     Failure/Error:
       expect(PushNotification::HttpPush).to receive(:deliver).once.
           with(PushNotification::EmailChangedEvent.new(user: user, email: new_email))

       (PushNotification::HttpPush (class)).deliver(#<PushNotification::EmailChangedEvent:0x00007fb899322658 @user=#<User id: 17588, created_at: "2021-04...code_digest: "", remember_device_revoked_at: nil, email_language: nil>, @email="hollis@example.org">)
           expected: 1 time with arguments: (#<PushNotification::EmailChangedEvent:0x00007fb899322658 @user=#<User id: 17588, created_at: "2021-04...code_digest: "", remember_device_revoked_at: nil, email_language: nil>, @email="hollis@example.org">)
           received: 0 times
```